### DISCUSSION TITLE
Beginner question: hotspot vs connect to Wifi

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#215] [Q&A]
I am just starting with LBB and am confused
I installed comitup, to have LBB start its own hotspot
Sure enough I get a WIFI little-backup-box-nnnn

the install instruction say:

** 2. Alternatively can reach the web UI of Little Backup Box when you are connected to its
*** hotspot as follows:
***
*** https://10.41.0.1 (secure, certificate can't be verified automatically, please confirm it)
*** http://10.41.0.1:8000 (insecure)

But when I join that and go to https://10.41.0.1. a popup window appears on my notebook to make me choose an existing network
and on the main browser window I get the SAME

But that is NOT what i want. I want to work in single hotspot (offline from the Internet) mode

How do I make it not ask for my local wifi network?

I only get the LBB UI on the insecure port 8000?

Also: can I change the SSID name?

**Comment by outdoorbits** [comment]
This looks like you didn't get http**s**://10.41.0.1/ but http://10.41.0.1/. Did your browser ask you to accept the certificate of lbb?
Which browser are you using?

**Comment by sejtam** [comment]
I am using latest Chrome on MAC right now
As soon as I open chrome while connceted to the lbb WIFI, it pops up a window 'Join 'little-backup-box-nnn') listing all SSIDS it sees. No request for me to accept a cert or such.

when I then go to https://10.41.0.1. i get the exact same

only when I go to the insecure http://10/41.0.1:8000  do I see the LBB menu

**Comment by outdoorbits** [comment]
I don't have any experience using mac. I'm using Firefox on Linux and 
Windows and this works well. Maybe someone else knows more...? WiFi 
connection is done by http://davesteele.github.io/comitup/ so maybe you 
get any mac specific information there? Please tell us!

**Comment by sejtam** [comment]
I tried installing again w/o comitup, but then it does not even provide 
its own hotspot.

I will need to figure out how rto disable the
walled garden popup that wants to offer to connect to an existing
Wifi. (and them make the *secure* port  provide the interface, rather
than the insecure one

**Comment by outdoorbits** [comment]
Of course, comitup is the hotspot. ;-)

**Comment by sejtam** [comment]
Yes, but it behaves contrary to what is claimed. 
1. It keeps popping up a walled garden dialog to select any of the existing wireless networks to connect to,
2. on the secure port. (https://10.41.0.1. it shows the same)
3. on the insecure port the menu is shown. (which is insecure as it is)
